### DISCUSSION TITLE
Definition of intraperitoneal injection

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1187] [Questions & Problems] [answered]
Dear team,

If I want to use an intraperitoneal injection, which organ tissue do I need to select in User defined? Is it just a matter of choosing the part that defines the intraperitoneal injection?
![image](https://user-images.githubusercontent.com/92715703/161375728-5002794d-5af4-4431-a167-20ba8088f719.png)

**Comment by HenrikCordes** [comment, marked as answer]
Dear @Carrie-nothing ,

An IP Administration is injected into the abdominal cavity. The capillaries from there drain into the portal vein.
With the IP route one can exclude strong delays in absorption processes and reduced bioavailability like after PO, but still have an first pass effect from the liver.

Thus I would choose the portal vein as input compartment and an formulation that allows to model potential small absorption delays. 

Maybe we get some more comments or recommendation from the community.

Hope this helps 
Henrik

**Reply by prvmalik** [reply]
Depends on the size of the molecule and its permeability. Most small molecules would not have an IP first pass because they would rapidly distribute in and around central organs. For most small molecule PBPK, dropping the drug into skin interstitial or muscle interstitial with a Ka through a formulation block or as an IV infusion is sufficient for IP admin.

For larger or non-permeable molecules the capillary flow could direct a strong portion to the portal vein. I've modeled 90% before, with 10% circulating through lymph.

For very large molecules, incomplete bioavailability with IP admin is also due to the molecules being essentially stuck at the local injection site. We see this with gold nanoparticles.

**Reply by Carrie-nothing** [reply]
thak you very much ,this helps me a lot!
Best wishes to you.
